pkgs/applications/version-management/got/default.nix

```diff
@@ -49,12 +49,15 @@ stdenv.mkDerivation (finalAttrs: {
 
   installCheckPhase = ''
     runHook preInstallCheck
-    test "$($out/bin/got --version)" = "${finalAttrs.pname} ${finalAttrs.version}"
+    test "$($out/bin/got --version)" = "got ${finalAttrs.version}"
```

Comment:
Mental note: there are version test templates in nixosTests.

Reply:
Do provide some context if you feel like it, I'm always happy to learn another aspect of nixpkgs that is new or unfamiliar to me :)

Reply:
1. Usually custom `installCheckPhase` is somewhat discouraged, since it is meant for running the upstream testsuite (e.g. [Zig has a test directory](https://github.com/ziglang/zig/tree/master/test)). More details in the [stdenv manual](https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/stdenv.chapter.md#the-installcheck-phase-ssec-installcheck-phase).

1. As the link above says, custom tests are better reserved to `passthru.tests`. 
  
   Indeed they are way more flexible since 
   1. they can rely on the entire Nix/Nixpkgs functionality, and 
   1. they run outside the build environment of the package, simulating a new installation.  

   As an example, I cite mine: one of the elvish tests runs an elvish shell script, and one of the live555 tests tries to build vlc. 

1. `testers.testVersion` is in the manual and shows some examples: https://nixos.org/manual/nixpkgs/unstable/#tester-testVersion